staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -318,6 +319,10 @@ type AuthorizationInfo struct {
 	Authorizer authorizer.Authorizer
 }
 
+func init() {
+	features.AddFeatureGates(utilfeature.DefaultMutableFeatureGate)
```

Comment:
other call sites seem to wrap in a utilruntime.Must?

Reply:
ah, done.